components/automate-ui/src/app/services/profiles/upload.service.spec.ts

```diff
@@ -8,26 +8,29 @@ class MockChefSessionService {
 }
 
 describe('UploadService', () => {
+  let service: UploadService;
+  let mockChefSessionService;
+
   beforeEach(() => {
-     this.mockChefSessionService = new MockChefSessionService();
-     this.service = new UploadService(this.mockChefSessionService);
+     mockChefSessionService = new MockChefSessionService();
+     service = new UploadService(mockChefSessionService);
   });
   describe('getProgress', () => {
     it('returns the correct object', () => {
-      expect(this.service.getProgress('file', 'percent', 'status', 'response'))
+      expect(service.getProgress('file', 25, 1, 'response'))
         .toEqual({
           'name': 'file',
-          'percent': 'percent',
-          'status': 'status',
+          'percent': 25,
+          'status': 1,
           'response': 'response'
```

Comment:
Well, so much for strict typing... these types were plain wrong, yet our current env does not complain.